cn_indian_payroll/cn_indian_payroll/report/epf_challan_report/epf_challan_report.py

```
@@ -15,39 +16,55 @@ def get_salary_slips(filters=None):
+        da_component = company_doc.custom_da_component
+    else:
+        frappe.throw("Company is a mandatory filter.")
 
-    # Fetch all Salary Slips based on conditions
     salary_slips = frappe.get_list(
         'Salary Slip',
         fields=["name", "employee", "custom_month", "custom_payroll_period", "company", "custom_statutory_grosspay"],
         filters=conditions,
-        order_by="name DESC",
-        limit_page_length=0  # Fetch all records
+        order_by="name DESC"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)